v2/memory.go

```diff
@@ -132,11 +60,11 @@ func (m *memoryController) Stat(g GroupPath, stats *statsv2.Metrics) error {
 		if err := sc.Err(); err != nil {
 			return err
 		}
-		key, v, err := parseKV(sc.Text())
+		filename, v, err := parseKV(sc.Text())
```

Comment:
It's not so clear, why `parseKV`(parse Key and value) function returns filename. I think `key` was way more clear.